In-house patch to Neutron's setup.py file.  It contains requirement
code which looks for the pyudev module.  Since this module is Linux
specific, we need to exclude it.  Patch has not yet been submitted
upstream.

--- quantum-2013.1.3/setup.py.orig   2013-09-19 10:57:28.784977728 -0600
+++ quantum-2013.1.3/setup.py    2013-09-19 10:57:48.050712916 -0600
@@ -24,6 +24,9 @@
     requires.append('pywin32')
     requires.append('wmi')
     requires.remove('pyudev')
+elif sys.platform == 'sunos5':
+    requires.remove('pyudev')
+    requires.remove('amqplib>=0.6.1')

 Name = 'quantum'
 Url = "https://launchpad.net/quantum"
